java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -198,7 +198,7 @@ public long getHostMemorySize() {
     }
   }
 
-  private final OffHeapState offHeap;
+  final OffHeapState offHeap;
```

Comment:
Who needs direct access to another columns off heap?  That feels like we are exposing too much of the internals and increasing coupling.

Reply:
Artifact of testing . will remove.

Reply:
Done.